lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -358,9 +358,7 @@ static void decomposeMixedModeDotOp(ModuleOp mod) {
     if (mmaLayout) {
       bool isNativeHopperFP8 =
           AElType.isFloat8E5M2() || AElType.isFloat8E4M3FNUZ();
-      bool isFP8 = isNativeHopperFP8 || AElType.isFloat8E5M2FNUZ() ||
```

Comment:
Float8E5M2FNUZ is another type that's still referenced in Utility.cpp.

Reply:
hmm true, I guess this one we never really used

Reply:
this type should basically never get here.

Reply:
Should we add an assert then?  (Or is it never used at all, in which case can we remove it from Utility.cpp?)